### ISSUE TITLE
Missing information for configuring app

### ISSUE THREAD

**Issue by jumpjack** [eclipsesource/tabris-plugin-firebase#76]
The [documentation ](https://github.com/firebase/quickstart-android/tree/master/messaging)does not clearly explain how to "register" a tabris app on firebase server to enable the app to receive notifications:

**Send to a single device
From Firebase console Notification section, click New Message.
Enter the text of your message in the Message Text field.
Set the target to Single Device.
Check the logs for the InstanceID token, copy and paste it into the Firebase console Token field.
If you cannot find the token in your logs, click on the LOG TOKEN button in the application and the token will be logged in logcat.
Click on the Send Message button.
If your application is in the foreground you should see the incoming message printed in the logs. If it is in the background, a system notification should be displayed. When the notification is tapped, the application should return to the quickstart application.**

Is the app registered by "token"? Is there a single token for each tabris app or is there a single token for developer app? How to access/generate these tokens?

Or does it register by its url/name? (com.my.app.whatelse)

There is no (more) "send message" in firebase console, there is "new campaign" with many variants and options, some screenshots would help.

**Comment by cookieguru** [comment]
> The [documentation ](https://github.com/firebase/quickstart-android/tree/master/messaging)does not clearly explain how to "register" a tabris app on firebase server to enable the app to receive notifications

It's the first link in the Readme you linked to

---

>  Is there a single token for each tabris app or is there a single token for developer app? How to access/generate these tokens?
https://firebase.google.com/docs/cloud-messaging/manage-tokens

From [the Firebase token docs](https://firebase.google.com/docs/cloud-messaging/manage-tokens): "On initial startup of your app, the FCM SDK generates a registration token for the client app instance".  Simply put, the token refers to an installed _instance_ of your app.

> There is no (more) "send message" in firebase console, there is "new campaign" with many variants and options, some screenshots would help.

The Firebase docs suggest using the [notifications composer](https://console.firebase.google.com/project/_/notification)--is that what you're trying?

**Comment by jumpjack** [comment]
Yes, I created a project, I associated an app (using a random package name):

![image](https://user-images.githubusercontent.com/1620953/166879932-cceb954c-26b7-4ca6-aad9-5f0996ae54c2.png)

 I created a campaign 

![image](https://user-images.githubusercontent.com/1620953/166879446-83614866-8384-4764-91a3-c6adfa7edb80.png)

using the token provided by [this ](https://github.com/eclipsesource/tabris-plugin-firebase/blob/master/doc/cloud-messaging.md) example source and inserted here:

![image](https://user-images.githubusercontent.com/1620953/166880237-4a2a7125-b1bb-4d09-93a7-6c3a31bf7f97.png)

But I am not sure at all of what I am doing, I am proceedingly randomly, documentation is quite focused on offline development, while I need to develop online, using playground and developer app.
Maybe I should use the package name of the developer app? Where do I find it?
